src/main/java/com/passpnu/passwordmanager/entity/Password.java

```diff
@@ -0,0 +1,26 @@
+package com.passpnu.passwordmanager.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
```

Comment:
Do not use `@Data` with JPA entities (https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/)

Reply:
So I need to add getters and setters manually